pages.ja/common/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+# docker compose
+
+> 複数コンテナを持つDockerアプリケーションの実行と管理をします。
+> 詳しくはこちら: <https://docs.docker.com/compose/reference/>.
+
+- 実行中のコンテナ全てをリスト表示する:
+
+`docker compose ps`
+
+- カレントディレクトリにある `docker-compose.yml`ファイルを使用してバックグラウンドで全てのコンテナを作成・起動する:
+
+`docker compose up -d`
```

Comment:
Long option found:

```suggestion
`docker compose up --detach`
```

Reply:
@Kumamoto-Hamachi If the above suggestion is applied, same must be updated in English pages too to be uniform.

Reply:
Thank you both for your reviews.
I will change the PR title to "docker-system, docker-compose: add Japanese translation, update page" and also change the docker-compose.md file for the English version as reviewed.